domains/gender-thief.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "SageStarCodes",
+           "email": "",
+           "discord": "139381119622578177",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.ogzcHCs16DVpRmFpEm6By2NtfO76stlyT7a8vKkevmox0ZDNWjl1-AA8UnrwvJbXe7rwyDQeID5JjHuuP29Ng4sCuk9hY1IZ4TPWrg-xCcI_KjP4mahLZVPj8B78p3_LIzudekBfmQZuiovmJ0YKOC2sXtpDE_3GW2JIHGry1hd5CjgyBhMcApMTXQDcekMQYJcVnAaXtwtPBLQREkf5DT3QFGdjHubWf97k1FecPZQsYaCHSP0VZ8ZivzQwzawMlGDHz89DDyt6dQcNpnWnvF9nbOjyeRrYY1XdpJt-j2epX5CXJp2bi4P-vH_loqC1ARJUkHIc9ONhcySstqccVA.RQP-sfFekMGt9ae-P6kqMg.r7wIznjbjJHPOdJmNtKfSWN2Cc-ILbdN_zYm0pMrzwJZpHtpjdZhWki6rG-Y0oozrR1OvHVgeSSV9IMeOBEkzFSDvKIAHyktQYZ9RT8bjpE.PfjG61w2kpz4gyaYVBb6Jg"
+        },
+    
+        "record": {
+            "CNAME": "fichna.sk"
```

Comment:
Error 523 (Origin is unreachable), please give a preview of your website.

Reply:
oh... right when my server crapped itself... well, whatever closing this no longer interested